_haikus/forkerino.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: forkerino
+author: page
+---
+
+On a sunny autumn day,
+Tried to fork a project.
+Terminal broke.
```

Comment:
Thanks for this poem. Please read the section about haikus in the `README.md` file of this repository. Haikus are poems that follow a specific format of three lines. They also have a certain number of syllables per line: 5 syllables in the first line, 7 syllables in the second line, 5 syllables in the last line. You can check your poem against haiku standards with [http://www.haikusyllablecounter.com/](http://www.haikusyllablecounter.com/).